coinlib_flutter/windows/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# The Flutter tooling requires that developers have a version of Visual Studio
+# installed that includes CMake 3.14 or later. You should not increase this
+# version, as doing so will cause the plugin to fail to compile for some
+# customers of the plugin.
+cmake_minimum_required(VERSION 3.14)
+
+# Project-level configuration.
+set(PROJECT_NAME "coinlib_flutter")
+project(${PROJECT_NAME} LANGUAGES CXX)
+
+# Invoke the build for native code shared with the other target platforms.
+# This can be changed to accommodate different builds.
+# add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../src" "${CMAKE_CURRENT_BINARY_DIR}/shared")
+# TODO re-enable the above line if/when the CMake process is made Windows-compatible.  See also below.
+
+# List of absolute paths to libraries that should be bundled with the plugin.
+# This list could contain prebuilt libraries, or libraries created by an
+# external build triggered from this build file.
+set(coinlib_flutter_bundled_libraries
+  # Defined in ../src/CMakeLists.txt.
+  # This can be changed to accommodate different builds.
+  # $<TARGET_FILE:coinlib_flutter>
+  # TODO re-enable the above line if/when the CMake process is made Windows-compatible.  See also above.
+  ${CMAKE_BINARY_DIR}/../../secp256k1.dll
```

Comment:
How is the library being picked up with this? Surely once it is built, the library would be in `${CMAKE_BINARY_DIR}/build/secp256k1.dll`?

Reply:
`CMAKE_BINARY_DIR` is `app/build/windows/x64`.  The DLL is copied into `app/build`.  There's probably another CMAKE variable available that refers to just `app/build`, but this works.

Reply:
OK thank. It will be good to verify this works with the test package. I assume everything is ready to test that? I might be able to publish a test release this weekend and I can see if there are a few others who can test.

Reply:
I considered moving the lib to CMAKE_BINARY_DIR but thought it would be better to remain consistent with coinlib conventions across platforms

Reply:
> OK thank. It will be good to verify this works with the test package. I assume everything is ready to test that? I might be able to publish a test release this weekend and I can see if there are a few others who can test.

Yes, everything is ready to test! 🥳